plugin/commands/me.md

```
@@ -367,12 +371,13 @@ Optional flag: `--from <path>` — copy from a local clone instead of fetching f
 
 7. **Print next steps.** One short summary to the user:
 
-   > `~/.me/ seeded from examples. Three files to personalize:`
+   > `~/.me/ seeded from examples. Four files to personalize:`
    > `  - identity.yaml — name, timezone, work, pets`
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)